File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] s = br.readLine().split(" ");
        int a,b,c,d,e,f;
        a=Integer.parseInt(s[0]);
        b=Integer.parseInt(s[1]);
        c=Integer.parseInt(s[2]);
        d=Integer.parseInt(s[3]);
        e=Integer.parseInt(s[4]);
        f=Integer.parseInt(s[5]);

        int x = 0;
        int y = 0;
        for (int i = -999; i<1000;i++) {
            for (int j=-999; j<1000;j++){
                if ((a*i+b*j==c) && (d*i+e*j==f)) {
                    x=i;
                    y=j;
                    break;
                }
            }
        }
        System.out.println(x+" "+y);
    }
}

File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.kt
fun main() {

    val arr = readLine()!!.split(" ").map { it.toInt() }
    val a = arr[0]
    val b = arr[1]
    val c = arr[2]
    val d = arr[3]
    val e = arr[4]
    val f = arr[5]


    for (x in -999..999) {
        for (y in -999..999) {
            if (a*x+b*y == c && d*x+e*y == f) {
                println("${x} ${y}")
                break
            }
        }
    }

}
